Source/HYPScatterPlot.m

```diff
@@ -506,29 +552,56 @@ - (void)drawPointsAndYThresholdsInRect:(CGRect)rect
         [self drawLineWithYValue:translatedY usingColor:self.averageYLineColor dashedStyle:YES];
     }
 
+    CGPoint selectedPoint = CGPointMake(CGFLOAT_MAX, CGFLOAT_MAX);
+    NSUInteger selectedIndex;
+
     for (NSInteger index = 0; index < numberOfPoints; index++) {
         CGPoint point = [self.dataSource pointAtIndex:index];
         CGPoint translatedPoint = CGPointMake(
-            [self translateValue:point.x srcMin:srcMinX srcMax:srcMaxX destMin:destMinX destMax:destMaxX],
-            [self translateValue:point.y srcMin:srcMinY srcMax:srcMaxY destMin:destMinY destMax:destMaxY]
-        );
+                                              [self translateValue:point.x srcMin:srcMinX srcMax:srcMaxX destMin:destMinX destMax:destMaxX],
```

Comment:
Mind dropping the abbreviations here?

+                                              [self translateValue:point.x srcMin:srcMinX srcMax:srcMaxX destMin:destMinX destMax:destMaxX],